nexus/external-api/src/lib.rs

```diff
@@ -1409,7 +1409,7 @@ pub trait NexusExternalApi {
         path_params: Path<params::AntiAffinityGroupPath>,
     ) -> Result<HttpResponseOk<ResultsPage<AntiAffinityGroupMember>>, HttpError>;
 
-    /// Fetch an anti-affinity group member
+    /// Fetch an anti-affinity group member (where that member is an instance)
```

Comment:
Note: https://github.com/oxidecomputer/omicron/pull/7650 will conflict here, be more brief with these comments

Reply:
Merged, marking resolved